### ISSUE TITLE
admission webhook "kubevirt-crd-status-validator.kubevirt.io" denied the request

### ISSUE THREAD

**Issue by wywself** [kubevirt/kubevirt#12732]
I want to update addvolume api, I don't need to pass in the disk, so I need to cancel the disk check. 

I update crds in /pkg/virt-operator/resource/generate/components/validations_generated.go, 
![image](https://github.com/user-attachments/assets/d7c96b35-45f7-4e32-88ea-bcda7db0e80a)

and I rebuild virt-operator, but it was not succeeded. the current error is as follows, do I need to modify anything else? Who can help me? Thank you.

api body:
```
{
    "name": "wyw-memsnap-pvc2",
    "volumeSource": {
        "persistentVolumeClaim": {
            "claimName": "wyw-memsnap-pvc2",
            "hotpluggable": true
        }
    }
}
```
return:
```
{
    "kind": "Status",
    "apiVersion": "v1",
    "metadata": {},
    "status": "Failure",
    "message": "admission webhook \"kubevirt-crd-status-validator.kubevirt.io\" denied the request: status.volumeRequests.addVolumeOptions.disk in body is required",
    "reason": "Invalid",
    "details": {
        "causes": [
            {
                "message": "status.volumeRequests.addVolumeOptions.disk in body is required"
            }
        ]
    },
    "code": 422
}
```

I expect return success.

**Comment by wywself** [comment]
If I update staging/src/kubevirt.io/client-go/api/openapi_generated.go, Which component do I need to make? Thank you.

**Comment by vladikr** [comment]
@wywself Hi, could you please explain how this volume will be used (consumed by the guest) if it's not attached to a `Disk`?

**Comment by mhenriks** [comment]
@wywself it sounds like you want to add the pvc to the virt-launcher pod but not make it available to the guest VM?  Not sure that is an operation we'd want to support

**Comment by wywself** [comment]
> @wywself it sounds like you want to add the pvc to the virt-launcher pod but not make it available to the guest VM? Not sure that is an operation we'd want to support

@mhenriks yes.Are there any plans to support it later? I plan to implement this feature.

**Comment by wywself** [comment]
> @wywself Hi, could you please explain how this volume will be used (consumed by the guest) if it's not attached to a `Disk`?

For example, For example, save the memory snapshot in this volume , etc.
